pkg/build/rpm.go

```diff
@@ -1,12 +1,50 @@
 package build
 
 import (
+	_ "embed"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
-func (b *Builder) getRPMFileNames(rpmSourceDir string) ([]string, error) {
+const (
+	modifyRPMScriptName = "10_rpm_install.sh"
+)
+
+//go:embed scripts/rpms/10_rpm_install.sh.tpl
+var modifyRPMScript string
+
+func (b *Builder) processRPMs() error {
+	rpmSourceDir, err := b.generateRPMPath()
+	if err != nil {
+		return fmt.Errorf("generating rpm path: %w", err)
+	}
+	// Only proceed with processing the RPMs if the directory exists
+	if rpmSourceDir == "" {
+		return nil
+	}
+	rpmDestDir := b.combustionDir
```

Comment:
I am not sure what use we get from moving the `b.combustionDir` property to `rpmDestDir` and using it in `copyRPMs`. Can't we just use `b.combustionDir` directly as a parameter to the `copyRPMs` function? That way we will save up one more line of code that we have to write without making the existing code too complex.

Reply:
I mainly added this so that it's clear that there was a variable that held where the rpms should go in the end, but like you said we can save that line by omitting it, will update 👍